datasets/conv_ai_2/README.md

```diff
@@ -0,0 +1,183 @@
+[Needs More Information]
```

Comment:
can you fill the yaml tags at the top of the dataset card please ?

More info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card